src/app/index.ts

```diff
@@ -0,0 +1,9 @@
+import Express from 'express';
+
+const PORT= 8080;
```

Comment:
I think we should use spaces before equal signs to improve readability.
E.g: 
```
const PORT = 8080
```

Reply:
of course.